src/python/library/tritonclient/grpc/_client.py

```diff
@@ -1466,6 +1490,8 @@ def wrapped_callback(call_future):
                 result = InferResult(response)
             except grpc.RpcError as rpc_error:
                 error = get_error_grpc(rpc_error)
+            except grpc.FutureCancelledError:
+                error = CancelledError()
```

Comment:
Why not just error = get_error_grpc(rpc_error) if `grpc.FutureCancelledError` is a well defined GRPC error?

Reply:
`grpc.FutureCancelledError` was missing debug_error_string() and causing some issues. I will look more closely into how to consume the error object.

Reply:
Not a big deal then IMO, but I think we should probably return an `InferenceServerException` with a message and CANCELLED code for consistency.

Reply:
Most grpc streaming client code today looks something like [this](https://github.com/triton-inference-server/server/blob/2b121fdd375556653ea3bcb9acb5eb08410bd094/qa/L0_decoupled/decoupled_test.py#L142-L145):
```python
if type(data_item) == InferenceServerException:
  raise data_item
else:
  response = data_item.get_response()
```
So I think if we'd break this flow by introducing a 3rd case of `CancelledError` when it hits `response = data_item.get_response()`